support-frontend/assets/helpers/forms/paymentIntegrations/payPalRecurringCheckout.ts

```diff
@@ -39,23 +42,24 @@ export type PayPalCheckoutDetails = {
 // ----- Functions ----- //
 function loadPayPalRecurring(): Promise<void> {
 	return new Promise((resolve) => {
-		const script = document.createElement('script');
-		script.onload = resolve;
+		const script: HTMLScriptElement = document.createElement('script');
+		script.onload = () => resolve;
```

Comment:
Should this be `script.onload = () => resolve();`, as we want to call `resolve` on `onload`?